### DISCUSSION TITLE
further intregration of native BTC support

### DISCUSSION THREAD

**Discussion by AltCoiFish** [DeFiCh/app#1083] [Ideas]
Currently the **SPV*** (Simplified Payment Verification) was introduced and implemented into the native wallet (https://github.com/DeFiCh/app).

This feature is currently "beta" and I would suggest to further develop the native integration of Bitcoin into DFI. 
We already interacting on some levels with the Bitcoin blockchain, using DFI anchoring. 
This is just an example and has nothing to do with my thread here, but worth mentioning. 

Also worth mentioning is, as Julian explicitly highlighted in his last Video, DFI was forked from Bitcoin. 
This is something we should focus on, to implement Bitcoin natively into DFI. 

**Current status-quo:**

You can send and receive Bitcoins to your DFI wallet using the Bitcoin network. 
![image](https://user-images.githubusercontent.com/79468797/144712893-c382bb04-adcf-4fc3-bc89-7ba1688c65f7.png)

But you can use it once your native Bitcoins are on DFI wallet, 
**you can not**

- swap them for DFI using Dex
- use them as collateral 
- use them for the BTC-DFI liquidity pool. 

If you want to have nativ BTC on DFI , you have to use CAKE (https://cakedefi.com/) sending your BTC there and exchanging it for DFI-Bitcoin "**dBTC**" . But then you can not send the BTC outside of the DFI network, you can use it as a "normal" token on the DFI chain. 

As much as I like Cake, but this is a central point of exchanging native BTC to dBTC and can not be done via Exchanges, etc. 

**What I would like to see:**
A development towards a possibility to exchange native BTC on the DEX or have some other interaction with native BTC on DFI chain.

**Comment by MX24C** [comment]
You are interested in the bridge between the BTC network and the DeFiChain, allowing to lock BTC for get dBTC (wrapped BTC on DeFiChain) and to unlock BTC by give dBTC back? With dBTC you can go on DEX for get DFI and vice versa.

**Reply by AltCoiFish** [reply]
>You are interested in the bridge between the BTC network and the DeFiChain, allowing to lock BTC for get dBTC (wrapped BTC on DeFiChain) and to unlock BTC by give dBTC back? With dBTC you can go on DEX for get DFI and vice versa.

correct, I would like to close the gap between BTC and dBTC, so we can freely trade into DFI without any restrictions. 
Would be great to lock BTC in a "vault-type" system and get dBTC back from the system, but it has to be native BTC I can send into this "vault"

**Reply by MX24C** [reply]
I do make the proposal to rename your proposal into "Non-custodial funds-transfer between DeFiChain and BTC network". Then this proposal is the twin to CFP 2111-16 in realization by Dr. Daniel Cagara/muirgalcier doing the same for the exchange between DeFiChain and the ETH-ecosystem.

With the bridge it will become possible to lock BTC for get dBTC (wrapped BTC) that can get used on DeFiChain and the opposite way to unlock BTC by returning dBTC.
